server/@types/express/index.d.ts

```diff
@@ -29,6 +29,7 @@ export type JourneyData = {
   csipRecord?: CsipRecord
   csipRecordCreated?: boolean
   hasValidationErrors?: boolean
+  journeyCompleted?: boolean
```

Comment:
can this new generic `journeyCompleted` property deprecate the `csipRecordCreated` above?